src/AppDelegate.cpp

```diff
@@ -213,7 +213,7 @@ void AppDelegate::toggleDomainServer(bool start) {
 
 void AppDelegate::toggleAssignmentClientMonitor(bool start) {
     if (start) {
-        _acMonitorProcess->start(QStringList() << "--min" << "4");
+        _acMonitorProcess->start(QStringList() << "--min" << "5");
```

Comment:
You're correct that with 4 we get to all assignments running "eventually" but it takes longer. Since we have 5 servers now, we should start with 5 so that the startup process is faster.

This change was not the source of the entity server problems. 



> On Nov 18, 2015, at 7:55 AM, Leonardo Murillo <notifications@github.com> wrote:
> 
> In src/AppDelegate.cpp:
> 
> > @@ -213,7 +213,7 @@ void AppDelegate::toggleDomainServer(bool start) {
> >  
> >  void AppDelegate::toggleAssignmentClientMonitor(bool start) {
> >      if (start) {
> > -        _acMonitorProcess->start(QStringList() << "--min" << "4");
> > +        _acMonitorProcess->start(QStringList() << "--min" << "5");
> Curious why this was a problem, @sethalves correct me if I'm wrong but a --min 5 simply means, keep AT LEAST this number of assignments running at all times, but would also keep a spare AC running which would be grabbed by the remaining assignment on queue and it would then just spawn a new spare?
> 
> Given the addition of the new assignment type and the previous information I considered valid, that would mean there was no change required as the --min 4 would simply start with 4 but then spawn a new one to match the number of queued assignments?
> 
> Also curious why Brad's list showed all assignments grabbed?
> 
> —
> Reply to this email directly or view it on GitHub.
>